### DISCUSSION TITLE
== Error ==  system call for:finished abnormally, OS return value: -9

### DISCUSSION THREAD

**Discussion by Steavnzhou** [ablab/spades#1186] [Q&A]
Hi,I'm running metaspades log failed and here are my spades.log and params.txt.
 
[spades.log](https://github.com/ablab/spades/files/12735522/spades.log)
[params.txt](https://github.com/ablab/spades/files/12735523/params.txt)

Thanks for your help.

**Comment by sahana24310-cell** [comment]
Did you get an answer for this issue?
